pkg/cmd/util.go

```diff
@@ -24,6 +24,8 @@ func AddCommonFlags(c *cobra.Command, cfg *api.Config) {
 		"DEPRECATED: Always pull the builder image even if it is present locally")
 	c.Flags().VarP(&(cfg.BuilderPullPolicy), "pull-policy", "p",
 		"Specify when to pull the builder image (always, never or if-not-present)")
+	c.Flags().Var(&(cfg.PreviousImagePullPolicy), "incremental-pull-policy",
+		"Specify when to pull the previous image for incremental builds (always, never or if-not-present)")
```

Comment:
none of this is necessary.

i'm setting cfg.PreviousImagePullPolicy in main.go, which ensures that when usage is printed, it prints the correct "default" value.